lte/protos/oai/BUILD.bazel

```diff
@@ -66,6 +90,12 @@ proto_library(
     deps = [":common_types_proto"],
 )
 
+python_proto_library(
+    name = "std_3gpp_types_python_proto",
+    protos = [":std_3gpp_types_proto"],
+    deps = [":common_types_python_proto"],
+)
+
```

Comment:
Out of curiosity: Why does this library has a dependency, while the below equivalent cpp library does not?
In most other cases this seems to be "symmetric".

Reply:
Obsolete